lib/participant.js

```diff
@@ -160,6 +211,30 @@ class Participant extends EventEmitter {
     return track;
   }
 
+
+  /**
+   * @private
+   */
+  _addTrackPublication(publication) {
+    const log = this._log;
+    if (this.trackPublications.has(publication.trackSid)) {
+      return null;
```

Comment:
Not sure how this is used, but just wondering: why return null instead of the existing publication?

Reply:
@markandrus This is how `._addTrack` also works today. A `null` implies that the publication already exists.